bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonSerializers.scala

```diff
@@ -69,7 +69,7 @@ object JsonSerializers {
   implicit val transactionReads: Reads[Transaction] = TransactionReads
   implicit val transactionOutPointReads: Reads[TransactionOutPoint] =
     TransactionOutPointReads
-  implicit val bitcoinFeeUnitReads: Reads[BitcoinFeeUnit] = BitcoinFeeUnitReads
+  implicit val bitcoinFeeUnitReads: Reads[BitcoinFeeRate] = BitcoinFeeUnitReads
```

Comment:
this isn't directly related to this PR, but this serializer is dangerous as it allows us to mis-type a fee unit given back from bitcoind i believe. Should this be typed as `Reads[SatoshisPerKiloByte]`